db/app/error-handling.js

```diff
@@ -0,0 +1,34 @@
+const express = require("express");
+
+const app = express();
+
+
+exports.errorHandling = () => {
```

Comment:
how are you going to access each error handling middleware function?

Reply:
I would save each one to a relevantly named function - handlePSQLErrors for example